docs/Installing_Required_Dependencies.md

```diff
@@ -0,0 +1,43 @@
+## Installing Required Dependencies
+
+When setting up a system for custom device development, we must ensure that all the necessary dependencies are installed.
+
+### Required Dependencies
+
+* LabVIEW
+* LabVIEW Real-Time (if you will be deploying the custom device to a Linux Real-Time target)
+
+The version of LabVIEW/LabVIEW Real-Time must match the major version (i.e. the year version) and bitness of VeriStand you will use to deploy the custom device. Refer to the following table for bitness compatibility.
+
+| VeriStand | LabVIEW |
+| --------- | ------- |
+| 2021 and later | 64-bit |
+| 2020 R6 and earlier | 32-bit |
+
+For example, to create a custom device for VeriStand 2021 R3, we could use LabVIEW 2021 64-bit or LabVIEW 2021 SP1 64-bit.

```

Comment:
We ran into some issues with using LabVIEW 2021 64-bit, so we only build with 2021 SP1 64-bit. I know this is just an example, but please only list that version here.